src/router/routes.js

```diff
@@ -48,6 +52,10 @@ const routes = [
       }
     ]
   },
+  {
+    path: '/404',
+    component: () => import('pages/404Error.vue')
+  },
   {
     path: '/:catchAll(.*)*',
     component: () => import('pages/404Error.vue')
```

Comment:
You can merge these two items using `alias`:

```suggestion
  {
    path: '/:catchAll(.*)*',
    alias: '/404',
    component: () => import('pages/404Error.vue')
  }
```